### DISCUSSION TITLE
Can't get the latest cardano wallet version v2022-04-27

### DISCUSSION THREAD

**Discussion by kamranali368** [cardano-foundation/cardano-wallet#3264] [Q&A] [answered]
Hi,
I have already installed the Cardano node and wallet in my ubuntu system. Now, I am updating my Cardano wallet to the latest v2022-04-27 in order to be compatible with the Cardano node 1.34.1 but I am not able to fetch the latest wallet version. 

Currently, I have a version v2022-01-18 and it is not able to sync with the newest version of the Cardano node. When I run the command TAG=$(git describe --tags --abbrev=0) && echo latest tag $TAG it shows latest tag v2022-01-18 not the v2022-04-27. Is there anything I am doing not correctly?

**Comment by piotr-iohk** [comment, marked as answer]
git describe description reads:
> The command finds the most recent tag that is reachable from a commit. If the tag points to the commit, then only the tag is shown.

So perhaps you're locally on the commit that belongs to  `v2022-01-18`. Try `git pull origin master` first.
